services/vm-watcher/vm_watcher_service.go

```diff
@@ -134,13 +132,13 @@ func (s Service) waitForChanges(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	logger.Info("got vsphere client")
+	logger.Info("Got vsphere client")
 
 	refs, err := s.vmFolderRefs(ctx)
 	if err != nil {
 		return err
 	}
-	logger.Info("got vm service folders", "refs", refs)
+	logger.Info("Got vm service folders", "refs", refs)
```

Comment:
Consider using lower visibility logs for these?

Reply:
No, I don't think so. These are not emitted often, only when the service starts or restarts due to a credential/rotation.